src/Drag_Drop.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-loop-func */
+/* eslint-disable no-restricted-syntax */
+// eslint-disable-next-line import/prefer-default-export
+export function dragDrop() {
+  const target = document.getElementById('ListContainer');
+  const items = target.getElementsByTagName('li');
+  let current = null;
+  for (const i of items) {
+    i.draggable = true;
+    // eslint-disable-next-line func-names
+    i.addEventListener('dragstart', function () {
+      current = this;
+    });
+
+    i.addEventListener('dragover', (evt) => {
+      evt.preventDefault();
+    });
+    // eslint-disable-next-line func-names
+    i.addEventListener('drop', function () {
+      if (this !== current) {
+        let currentpos = 0; let
+          droppedpos = 0;
+        // eslint-disable-next-line no-plusplus
+        for (let it = 0; it < items.length; it++) {
+          if (current === items[it]) { currentpos = it; }
+          if (this === items[it]) { droppedpos = it; }
+        }
+        if (currentpos < droppedpos) {
+          this.parentNode.insertBefore(current, this.nextSibling);
+        } else {
+          this.parentNode.insertBefore(current, this);
+        }
+      }
+    });
+  }
+}
```

Comment:
- [ ] The issue your project is missing an array of object that will render dynamically to the DOM when I load the page currently nothing is being rendered when I load the page.. please add them again because am sure you worked on that in the previous project
- [ ] You need to add another file that should contain methods related to the status updates (`completed`: `true` / `false`).
- [ ] After getting list of todos each list item is supposed to have an event listeners (`dragstart`, `dragover`, `drop`) and to checkbox (`change`).
- [ ] The task object's values for `index` and `completed` are updated upon user actions.

Please work on that!